pkg/kubectl/cmd/get_test.go

```diff
@@ -135,114 +133,39 @@ func TestGetUnknownSchemaObject(t *testing.T) {
 	cmd.SetOutput(buf)
 	cmd.Run(cmd, []string{"type", "foo"})
 
-	expected := cmdtesting.NewInternalType("", "", "foo")
-	actual := tf.Printer.(*testPrinter).Objects[0]
-	if !reflect.DeepEqual(expected, actual) {
-		t.Errorf("unexpected object: %#v", actual)
-	}
-	if buf.String() != fmt.Sprintf("%#v", expected) {
-		t.Errorf("unexpected output: %s", buf.String())
-	}
-}
-
-// Verifies that schemas that are not in the master tree of Kubernetes can be retrieved via Get.
-// Because api.List is part of the Kube API, resource.Builder has to perform a conversion on
-// api.Scheme, which may not have access to all objects, and not all objects are at the same
-// internal versioning scheme. This test verifies that two isolated schemes (Test, and api.Scheme)
-// can be conjoined into a single output object.
-//
-// The expected behavior of the `kubectl get` command is:
-// 1. objects using unrecognized schemes will always be returned using that scheme/version, "unlikelyversion" in this test;
-// 2. if the specified output-version is a recognized, valid Scheme, then the list should use that scheme, and otherwise it will default to the client version, registered.GroupOrDie(api.GroupName).GroupVersion.String() in this test;
-// 3a. if the specified output-version is a recognized, valid Scheme, in which the requested object (replicationcontroller) can be represented, then the object should be returned using that version;
-// 3b. otherwise if the specified output-version is unrecognized, but the requested object (replicationcontroller) is recognized by the client's codec, then it will be converted to the client version, registered.GroupOrDie(api.GroupName).GroupVersion.String() in this test.
-func TestGetUnknownSchemaObjectListGeneric(t *testing.T) {
```

Comment:
We no longer do anything with `--output-version`.  I'd like to leave disconnecting/doc-ing that to a followup, but this this makes us generic for basic CRUD.